tests/datasets/test_user_info.py

```diff
@@ -110,3 +111,25 @@ def test_dapla_lab_user_info_short_email_no_email_in_jwt(
 ):
     monkeypatch.setenv("OIDC_TOKEN", fake_jwt)
     assert DaplaLabUserInfo().short_email is None
+
+
+@pytest.mark.parametrize(
+    ("environment_variable_name", "environment_variable_value", "expected_team"),
+    [
+        (DAPLA_GROUP_CONTEXT, "dapla-metadata-developers", "dapla-metadata"),
+        (
+            DAPLA_GROUP_CONTEXT,
+            "dapla-metadata-data-admins",
+            "dapla-metadata",
+        ),
+    ],
+)
```

Comment:
I feel that we could add a few more test cases, for example:

- The environment variable isn't set
- Malformed value
- Empty value